examples/tutorials/self-paced-training/part-5_federated_learning_applications_in_industries/chapter-11_federated_learning_in_healthcare_lifescience/11.2_drug_discovery/11.2.2_drug_discovery_amplify/finetuning_amplify.ipynb

```diff
@@ -0,0 +1,373 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f1854999-9b80-4277-ae4c-c8d8c979b3b8",
+   "metadata": {},
+   "source": [
+    "# Federated Fine-tuning of an AMPLIFY Model\n",
+    "\n",
+    "This example demonstrates how to use the AMPLIFY protein language model from [chandar-lab/AMPLIFY](https://github.com/chandar-lab/AMPLIFY) for fine-tuning on multiple downstream tasks. AMPLIFY is a powerful protein language model that can be adapted for various protein-related tasks. In this example, we'll show how to fine-tune AMPLIFY for the prediction of several protein properties using antibody sequence data. For more details, please refer this [paper](https://www.biorxiv.org/content/10.1101/2024.09.23.614603v1)."
+   ]
```

Comment:
The training script is based on regular pytorch (as compared with BioNeMo)?

Reply:
Good point. Added some info "Note, this script assumes a regular Pyhton environment and doesn't rely on running Docker as in the previous example. For running AMPLIFY within the BioNeMo Framework, please see [here](https://docs.nvidia.com/bionemo-framework/latest/models/amplify/)."